engine/samples/hello-cubos/main.cpp

```diff
@@ -0,0 +1,45 @@
+#include <cubos/engine/cubos.hpp>
+
+/// [Hello Cubos]
+void sayHelloCubos()
+{
+    CUBOS_INFO("Hello CUBOS");
+}
+/// [Hello Cubos]
+
+/// [Hello World]
+void sayHello()
+{
+    CUBOS_INFO("Hello");
+}
+
+void sayWorld()
+{
+    CUBOS_INFO("World");
+}
+/// [Hello World]
+
+int main()
+{
+    /// [Engine]
+    cubos::engine::Cubos cubos;
```

Comment:
```suggestion
    Cubos cubos{};
```

If you don't add the braces the constructor doesn't run - which means undefined behaviour.
I love C++ :)

Reply:
I didn't change that. Was this sample not running before? lmao

Reply:
Its UB - doesnt mean it doesnt work, justq that it may stop working whenever the compiler decides so